README.md

```diff
@@ -1,17 +1,120 @@
-# Front-End Repo for Node/React Practicum
+## Project Overview
+**Name**: Node/React Practicum Front-End  
```

Comment:
When we finalize a name for the app, we'll need to update this

Reply:
Got it!